src/clip_search/vg_objects_generate.py

```diff
@@ -0,0 +1,124 @@
+import sys
+sys.path.append("..")
+sys.path.append(".")
+from services.data_utilizer import DataUtilizer
+from services.vlm_models_service import SearchModel
+from clip_search.evaluate_utils import evaluate_objects
+from services.settings import settings
+import os
+import pandas as pd 
+import torch
+import spacy 
+from tqdm import tqdm 
+from transformers import PegasusForConditionalGeneration, PegasusTokenizer
```

Comment:
since this is not used, why import it?

Reply:
It is actually used for an experiment for rephrasing. Now it is deleted.